large-message-core/src/main/java/com/bakdata/kafka/NoBlobStorageClient.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 bakdata
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.bakdata.kafka;
+
+class NoBlobStorageClient implements BlobStorageClient {
```

Comment:
If no base path has been configured, the serde still works as long as no message exceeds the max byte size. If a message exceeds the maximum size, this client would then be called an exceptions would be thrown. https://github.com/bakdata/kafka-s3-backed-serde/blob/fbbb13c8aa46b568c58992c6809442031e32035b/large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java#L196

Reply:
Ok, unless I'm missing something that seems kinda dangerous to me. If someone forgets to configure the base path, they won't notice it until the first message exceeds the max size. And even then, the connection of an `UnsupportedOperationException` thrown in some internal class to a missing base path is all but obvious.

I'd suggest making that configuration either explicit (e.g., `large.message.backing.enabled=false`) or log a warning that the base path is missing and we fall back to this client.

Reply:
Ok, actually it would already fail here: https://github.com/bakdata/kafka-s3-backed-serde/blob/feature/support-azure-blob-storage/large-message-core/src/main/java/com/bakdata/kafka/LargeMessageStoringClient.java#L107 and it had been like this before. The nice benefit of this behavior is that if the deserialization exception handler is configured as LogAndContinueExceptionHandler, the message would be just ignored by Kafka. But of course, I can add a log when using the NoBlobStorageClient

Reply:
What do you think of https://github.com/bakdata/kafka-s3-backed-serde/pull/17/commits/756d2f410df8c0a13ce3f9fa11766a59c25c2969 ?